src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class DeadlineTest {
+    @Test
+    public void getDetails_newDeadlineCreated_returnStringArray() {
```

Comment:
I like how you follow the alternative naming convention for testing methods here!